### ISSUE TITLE
PTE-gated faults incorrectly said to depend on PCC?

### ISSUE THREAD

**Issue by nwf-msr** [riscv/riscv-cheri#107]
The PTE extension seems confused, gating store fault decisions on PCC rather than the authorizing capability presented to store instructions (that is, on `cs1` in `sc cs2, offset(cs1)`; see https://github.com/riscv/riscv-cheri/blob/ce6d9ce0fa4dc1b0ab74e6d83be618089430cd40/src/insns/store_32bit_cap.adoc).  Specifically, all three of these sentences seem dubious to me:
 
* https://github.com/riscv/riscv-cheri/blob/ce6d9ce0fa4dc1b0ab74e6d83be618089430cd40/src/cheri-pte-ext.adoc?plain=1#L38-L39

* https://github.com/riscv/riscv-cheri/blob/ce6d9ce0fa4dc1b0ab74e6d83be618089430cd40/src/cheri-pte-ext.adoc?plain=1#L54-L56

* https://github.com/riscv/riscv-cheri/blob/ce6d9ce0fa4dc1b0ab74e6d83be618089430cd40/src/cheri-pte-ext.adoc?plain=1#L58-L59

**Comment by jrtc27** [comment]
Agreed. PCC shouldn't have store permission, let alone store capability.
